RELEASE_NOTES.md

```diff
@@ -1,11 +1,5 @@
 ## Enhancements
-
-- [KOGITO-2562](https://issues.redhat.com/browse/KOGITO-2562) - Knative Eventing resources created for Kogito Applications that uses [Knative Addon](https://github.com/kiegroup/kogito-examples/tree/stable/process-knative-quickstart-quarkus).
```

Comment:
Why are all these links removed?

Reply:
Based on @ricardozanini's [commit](https://github.com/kiegroup/kogito-cloud-operator/commit/3f0fc09a234845f615ff85cba1d6453efceb2518#diff-68d24fa81558aae3d8c59e2aa57a4fa719ea3b04d7fa14beff45f16f00858f50), I believe the intention is to update the document for your changes only.

Reply:
This issue was already solved on 0.17, now we are releasing 1.0.0

Reply:
Meaning that this document should reflect only changes for the next release.

Reply:
@Kevin-Mok don't remove these lines, only the first one is ok. The rest should be included in this release.

Reply:
I think that the commit you mention removed JIRAs which were addressed in already released (or going to be released) version. The KOGITO-2562 is addressed in 0.17, but the JIRAs below are for 1.0